src/main/java/org/opentestsystem/ap/imrt/common/model/Stimulus.java

```diff
@@ -0,0 +1,30 @@
+package org.opentestsystem.ap.imrt.common.model;
+
+import javax.persistence.CascadeType;
+import javax.persistence.DiscriminatorValue;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.JoinColumn;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+import java.util.List;
+
+@Entity
+@Table(name = "item")
+@DiscriminatorValue("stim")
+public class Stimulus extends BaseItem {
+    @JoinColumn(name = "item_key_stim")
+    @OneToMany(
+            cascade = {CascadeType.ALL},
+            fetch = FetchType.LAZY
+    )
+    private List<StimLink> relatedItems;
```

Comment:
I realize you are using a common data model for this but it seems odd that these are `StimLink`s. Because they are really `ItemLink`s aren't they?

Reply:
On a more general level I agree that it is an item link - it links one item to another.  In this case I think the intent here is to link a Stimulus to an Item (which is the only item association I'm aware of in the SB ecosystem).  From that perspective, I think it makes sense to call it a stim link (although I'd probably spell it out to be `StimulusLink`).